### DISCUSSION TITLE
Welcome to Adam image Discussions!

### DISCUSSION THREAD

**Discussion by eduardofilo** [eduardofilo/RG350_adam_image#77] [Announcements]
## Welcome
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by SuperBadger77** [comment]
Have you thought about a Discord server?

**Reply by eduardofilo** [reply]
Too much buzz for me. I try to minimize social networks activity, even if it seems otherwise. If I like github systems it is because it notifies by mail that it is my favorite communication system, so I manage the issues and now this discussions in that way, which is more bearable for me. I'm on the usual Discord servers, but it's hard for me to keep up with the conversations, especially since last month. I stop complaining. :-)

**Reply by eduardofilo** [reply]
@SuperBadger77 Just by curiosity, how did you realize so quickly that the Discussions module had been activated in the repository? I am interested to know if the participants in the issues are notified in any way, for example.

**Comment by SuperBadger77** [comment]
Ha! I've got notifications (email alerts and push) set up for the Githubs I'm interested in. On this one it's set to everything - I'm a keen follower of the Adam image :-)

**Reply by eduardofilo** [reply]
🤣🤣🤣
Thanks @SuperBadger77. I hope I do not disappoint your trust.
